build_for_ultimaker.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
-
+#
+# Copyright (C) 2018 Ultimaker B.V.
+# Copyright (C) 2018 Olliver Schinagl <oliver@schinagl.nl>
+# Copyright (C) 2018 Raymond Siudak <raysiudak@gmail.com>
+#
+# SPDX-License-Identifier: AGPL-3.0+
```

Comment:
its easier on the eyes to have 1 blank line between the end of the comment block and the first line of code